tests/bats/tests.bats

```diff
@@ -138,3 +146,24 @@ function create_tap() {
 @test "tcp server hw" {
   skip "not implemented"
 }
+
+@test "file rename spt" {
+  skip "Not working in rumprun (see issue XXX)"
+  dd if=/dev/zero of=data.ext2 count=1024 bs=1024
+  mkfs.ext2 data.ext2
+  run ${TIMEOUT} 30s ${SOLO5_SPT} --disk=data.ext2 file_rename_test.bin '{"cmdline":"test_rename /test","blk":{"source":"etfs","path":"/dev/ld0a","fstype":"blk","mountpoint":"/test"}}'
+  echo "$output"
```

Comment:
remove debug statement? or do `echo "$output" >&2` so that it only shows the output on test failure?

Reply:
Actually bats does that, it will print it just in case of failure. Also, without the `echo "$output"` the output is not printed for failures.